devicetypes/qubino/qubino-flush-dimmer.src/qubino-flush-dimmer.groovy

```diff
@@ -0,0 +1,890 @@
+	// Handles all Z-Wave commands we aren't interested in
+	log.debug("zwaveEvent other commands: ${cmd}")
+	[:]
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd, ep = null) {
+	log.debug "BasicReport: ${cmd}"
+	dimmerEvents(cmd)
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicSet cmd, ep = null) {
+	log.debug "BasicSet: ${cmd}"
+	dimmerEvents(cmd)
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.switchmultilevelv3.SwitchMultilevelReport cmd, ep = null) {
+	/*
+	This cmd value reports inaccurate values which often differ from the values reported by SwitchMultilevelSet by 1-3%.
+	Probbably we should only depend on SwitchMultilevelSet report.
+	*/
+	log.debug "SwitchMultilevelReport: ${cmd}"
+	//dimmerEvents(cmd)
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.switchmultilevelv3.SwitchMultilevelSet cmd, ep = null) {
+	log.debug "SwitchMultilevelSet: ${cmd}"
+	dimmerEvents(cmd)
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.switchmultilevelv3.SwitchMultilevelStartLevelChange cmd) {
+	log.debug "SwitchMultilevelStartLevelChange: ${cmd}"
+	def result = []
+	result << createEvent(name: "switch", value: "on")
+	result << createEvent(name: "level", value: cmd.startLevel)
+	return result
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.switchmultilevelv3.SwitchMultilevelStopLevelChange cmd) {
+	log.debug "SwitchMultilevelStopLevelChange: ${cmd}"
+	refresh()
+}
+
+private dimmerEvents(physicalgraph.zwave.Command cmd, ep = null) {
+	def value = (cmd.value ? "on" : "off")
+	def result = [createEvent(name: "switch", value: value)]
+	if (cmd.value <= 100) {
+		result << createEvent(name: "level", value: cmd.value == 99 ? 100 : cmd.value)
+		if (cmd.value != 0) {
+			state.lastDimmingLevel = cmd.value
+			log.debug "Dimming level state set to ${cmd.value}"
+		}
+	}
+	log.debug "dimmerEvents: ${result}"
+	return result
+}
+def zwaveEvent(physicalgraph.zwave.commands.meterv3.MeterReport cmd, ep = null) {
+	log.debug "MeterReport: ${cmd}"
+	handleMeterReport(cmd)
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.sensormultilevelv5.SensorMultilevelReport cmd, ep = null) {
+	log.debug "SensorMultilevelReport: ${cmd}, endpoint: ${ep}"
+	createChildEvent(cmd)
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cmd, ep = null) {
+	log.debug "NotificationReport: ${cmd}, endpoint: ${ep}"
+	createChildEvent(cmd)
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.sensorbinaryv2.SensorBinaryReport cmd, ep = null) {
+	log.debug "SensorBinaryReport: ${cmd}, endpoint: ${ep}"
+	def result
+	if (cmd.sensorValue == 0) {
+		result = createEvent([ name: "sensor", value: "inactive", descriptionText: "endpoint: ${ep}" ])
+	} else if (cmd.sensorValue == 255) {
+		result = createEvent([ name: "sensor", value: "active", descriptionText: "endpoint: ${ep}" ])
+	}
+	return result
+}
+
+def zwaveEvent(physicalgraph.zwave.commands.manufacturerspecificv2.ManufacturerSpecificReport cmd) {

```

Comment:
If we're not using this info anywhere, I think it is unnecessary to define this handler